be/src/olap/cumulative_compaction.cpp

```diff
@@ -452,9 +453,6 @@ OLAPStatus CumulativeCompaction::_do_cumulative_compaction() {
     // 6. delete delta files which have been merged into new cumulative file
     _delete_unused_delta_files(&unused_rowsets);
```

Comment:
rename this function name to _delete_unused_rowset?

Reply:
OK